addons/pkg/util/addon_util.go

```diff
@@ -79,6 +82,11 @@ func GenerateAppNamespaceFromAddonSecret(addonSecret *corev1.Secret, defaultAddo
 	return defaultAddonNamespace
 }
 
+// GenerateDataValueSecretNameFromAddonAndClusterNames generates data value secret name from addon names
+func GenerateDataValueSecretNameFromAddonNames(clusterName, addonName string) string {
```

Comment:
nit: Could we just name it ```GenerateDataValueSecretName``` ?

Reply:
+1 but with package name is passed in. We should try to to use updated terminology(package) in functions names as much as possible.

Reply:
Renamed to `GenerateDataValueSecretName ` to match @maralavi 's PR